packages/client/src/declarations/index.ts

```diff
@@ -1029,6 +1029,7 @@ function requestWithStateWrapper(
         await fetchAllDuplicateDeclarations(data.data as Query)
       }
       await fetchAllMinioUrlsInAttachment(data.data as Query)
+      await fetchAllMinioUrlsForSignature(data.data as Query)
```

Comment:
The way this would work is that it will download all the attachments first, wait for all of them to be finished, then start downloading all the signatures. It would be faster if we remove the waterfall here no?

Reply:
What we can do is, from these two functions get the urls back into an array, then fetch all the urls in a single go